### ISSUE TITLE
npm install eslint - enoent ENOENT: no such file or directory

### ISSUE THREAD

**Issue by qgreg** [eslint/eslint#13901]
I'm trying to install https://github.com/actions-on-google/dialogflow-name-psychic-nodejs for my own learning and edification. I'm new to nodejs, so I may be missing something obvious. I appreciate your time and assistance

.

I had encountered errors with installing dependancies previously, so I decided to remove them all from the default package.json and add them in one by one. This may be foolish on my part. Do let me know. Today I started with the following:
 
<details>
<summary>package.json</summary>:

```json
{
  "name": "name-psychic",
  "description": "Have your mind read on the spot",
  "version": "0.0.1",
  "private": true,
  "license": "Apache Version 2.0",
  "author": "Google Inc.",
  "engines": {
    "node": "12"
  },
  "scripts": {
    "lint": "eslint --fix \"**/*.js\"",
    "start": "firebase serve --only functions",
    "deploy": "firebase deploy --only functions",
    "test": "npm run lint"
  }
}
```
</details>

<details>
<summary>Environment</summary>
Here is my environment:
* Mac OS 10.15.7
* VirtualBox 6.1.14
* vagrant 2.2.10
* Ubuntu 18.04.5 LTS (GNU/Linux 4.15.0-122-generic x86_64)
</details>

* **ESLint Version:** 7.14.0
* **Node Version:** 12.20.0
* **npm Version:** 6.14.8

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**
I don't know. Is this relevant to this issue? I welcome assistance and clarification.

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

Without the install, I am not sure this is relevant. 
```js

```
</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash
npm install eslint --save-dev
```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**
The output is as follows:
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /vagrant/chin-whistle/dialogflow-name-psychic-nodejs/functions/node_modules/esrecurse/node_modules/estraverse/package.json.1140522508
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/vagrant/chin-whistle/dialogflow-name-psychic-nodejs/functions/node_modules/esrecurse/node_modules/estraverse/package.json.1140522508'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2020-12-02T18_21_49_401Z-debug.log

Here is the full log: [err.log](https://github.com/eslint/eslint/files/5631604/err.log)

Thank you so much.

**Comment by anikethsaha** [comment]
Thanks for the issue. 

Please remove the node module and the lock file and try adding the dependencies one by one or add them in `package.json` and then run `npm  i` to install them all

**Comment by qgreg** [comment]
Thanks for your response. As the error above was from  `npm i` followed by  `npm install eslint --save-dev`, I tried this package.json that included just the eslint dependency:
```json
{
  "name": "name-psychic",
  "description": "Have your mind read on the spot",
  "version": "0.0.1",
  "private": true,
  "license": "Apache Version 2.0",
  "author": "Google Inc.",
  "engines": {
    "node": "12"
  },
  "scripts": {
    "lint": "eslint --fix \"**/*.js\"",
    "start": "firebase serve --only functions",
    "deploy": "firebase deploy --only functions",
    "test": "npm run lint"
  },
  "devDependencies": {
    "eslint": "^7.14.0"
  }
}
```

Which delivered this error:
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /vagrant/chin-whistle/dialogflow-name-psychic-nodejs/functions/node_modules/chalk/node_modules/color-name/package.json.3946450852
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/vagrant/chin-whistle/dialogflow-name-psychic-nodejs/functions/node_modules/chalk/node_modules/color-name/package.json.3946450852'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vagrant/.npm/_logs/2020-12-04T20_30_04_911Z-debug.log

Here is the above listed log.
[err2.log](https://github.com/eslint/eslint/files/5645312/err2.log)

I will admit that I don't I understand what troubleshooting strategy I just accomplished or what is going on. I appreciate the help.
